server/api/test/index.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import request from 'superagent';
+import supertest from 'supertest';
+
+/**
+ * Internal dependencies
+ */
+import { allowNetworkAccess } from 'test/helpers/nock-control';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'api', function() {
+	let app, config, localRequest, sandbox;
+
+	allowNetworkAccess();
+	useSandbox( newSandbox => sandbox = newSandbox );
+
+	before( () => {
+		config = require( 'config' );
+		sandbox.stub( config, 'isEnabled' ).returns( true );
+		app = require( '../' )();
+		localRequest = supertest( app );
+	} );
+
+	afterEach( function() {
+		sandbox.restore();
+	} );
+
+	it( 'should return package version', function( done ) {
+		const version = require( '../../../package.json' ).version;
+
+		localRequest.get( '/version' )
+			.expect( 200, { version: version }, done );
+	} );
+
+	it( 'should clear oauth cookie and redirect to login_url', function( done ) {
+		localRequest
+			.get( '/logout' )
+			.redirects( 0 )
+			.set( 'cookie', 'wpcom_token=test' )
+			.expect( 'location', config( 'login_url' ) )
+			.expect( 'set-cookie', 'wpcom_token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT' )
+			.expect( 302, done );
+	} );
+
+	if ( process.env.NODE_ENV === 'desktop' ) {
```

Comment:
This one is tricky. We need to keep this check unless we mock all desktop specific config settings and whatever else we need to run it in `test` environment.